test/end-to-end/vehicle.js

```diff
@@ -457,6 +461,18 @@ test('vehicle request - override auth header', async(t) => {
   });
 });
 
+test('vehicle request - get charge limit', async(t) => {
+  const response = await t.context.kia.getChargeLimit();
+
+  t.is(typeof response.limit, 'number');
+});
+
+test('vehicle request - set charge limit', async(t) => {
+  const response = await t.context.kia.setChargeLimit(0.9);
+
+  t.is(response.status, 'success');
+});
```

Comment:
I thought about adding another test which checks the validation for limit works as intended but that seems like it would test the API, not the SDK.

Reply:
I agree, no need to test the validation limit.